test/e2e/main_test.go

```diff
@@ -58,7 +58,7 @@ const (
 )
 
 var (
-	e2eConfig   = config.NewFromFlags()
+	environment = config.NewFromFlags()
```

Comment:
Should we rename the package to `environment`? Or maybe `config.NewEnvironmentFromFlags`?

Reply:
I tried renaming it to `env` but it obviously conflicted with the one from the `e2e-framework`; `environment` would mean we have to find another name for the global variable, which I couldn't find a better name for atm; I like `config.NewEnvironmentFromFlags`, I'll do that 🙏